pyccel/epyccel.py

```diff
@@ -63,8 +63,11 @@ def epyccel_seq(function_or_module,
     if isinstance(function_or_module, FunctionType):
         pyfunc = function_or_module
         code = get_source_function(pyfunc)
-        tag = random_string(8)
-        module_name = 'mod_{}'.format(tag)
+
+        module_name = random_string(prefix='mod_', n=8)
+        while module_name in sys.modules.keys():
+            module_name = random_string(prefix='mod_', n=8)
```

Comment:
As a follow up to my previous comment, with the original `random_string` function this would read
```
module_name = 'mod_' + random_string(8)
while module_name in sys.modules.keys():
    module_name = 'mod_' + random_string(8)
```
which is totally equivalent and just as clear.